doc/src/manual/functions.md

```diff
@@ -140,6 +146,33 @@ Int8
 This function will always return an `Int8` regardless of the types of `x` and `y`.
 See [Type Declarations](@ref) for more on return types.
 
+Incidentally, the type of the arguments can also be specified similarly,
```

Comment:
I think this is a bit confusing since the behavior is not so similar; for example there is no conversion and return types don't affect dispatch.

Reply:
Here the "similarity" I mentioned was more superficial, just about the syntax (i.e. the same `::` token). Now, I see several options to address your concern.
1. Removing the paragraph. I added it because I thought it was a good think to connect related topics (thus the start with "Incidentally"), but you may disagree.
2. Make the adverb "similarly" sound more specific: Ex.: “Incidentally, the type of the arguments can also be specified in the signature with a similar syntax, but [...]”
3. Remove the adverb. Ex: “Incidentally, the type of the arguments can also be specified in the signature, but [...]”
@JeffBezanson, what do you prefer?

Reply:
I would just remove it.

Reply:
Ok, done!